### PR TITLE
Added solution

### PR DESCRIPTION
**Pull request by PavloSich**
[DEMO LINK](https://kapidastra.github.io/react_tabs-js/)

### CODE REVIEW

File: src/App.jsx [hunk 1]
```diff
@@ -1,44 +1,39 @@
...
-          </li>
-        </ul>
-      </div>
+      <Tabs
+        tabs={tabs}
+        activeTabId={activeTabId}
+        onTabSelected={handleTabSelected}
+      />
 
       <div className="block" data-cy="TabContent">
```
Comment (nazarmatsevych) [inline src/App.jsx:34]: Move this block to tabs component
image

**Review by nazarmatsevych — CHANGES_REQUESTED** [review]
Almost done, please fix the comment below

File: src/components/Tabs/Tabs.jsx [hunk 2]
```diff
@@ -1 +1,36 @@
...
+  const currentTab = tabs.find(tab => tab.id === activeTabId) || tabs[0];
+
+  return (
+    <div data-cy="TabsComponent">
+      <div className="tabs is-boxed">
+        <ul>
+          {tabs.map(tab => (
+            <li
+              key={tab.id}
+              className={tab.id === activeTabId ? 'is-active' : ''}
```
Comment (etojeDenys) [inline src/components/Tabs/Tabs.jsx:13]:
```suggestion
              className={tab.id === currentTab.id ? 'is-active' : ''}
```

File: src/components/Tabs/Tabs.jsx [hunk 3]
```diff
@@ -1 +1,36 @@
...
+              data-cy="Tab"
+            >
+              <a
+                href={`#${tab.id}`}
+                data-cy="TabLink"
+                onClick={() => {
+                  if (tab.id !== activeTabId) {
```
Comment (etojeDenys) [inline src/components/Tabs/Tabs.jsx:20]:
```suggestion
                  if (tab.id !== currentTab.id) {
```

**Review by etojeDenys — CHANGES_REQUESTED** [review]
it should fix the tests

File: src/App.jsx [hunk 4]
```diff
@@ -1,44 +1,35 @@
...
 
-          <li data-cy="Tab">
-            <a href="#tab-2" data-cy="TabLink">
-              Tab 2
-            </a>
-          </li>
+  return (
+    <div className="section">
+      <h1 className="title">
+        Selected tab is {tabs.find(tab => tab.id === activeTabId).title}
```
Comment (volodymyr-soltys97) [inline src/App.jsx:25]: Move this logic from jsx to the helper variable and use it here

File: src/components/Tabs/Tabs.jsx [hunk 5]
```diff
@@ -1 +1,36 @@
...
+  const currentTab = tabs.find(tab => tab.id === activeTabId) || tabs[0];
+
+  return (
+    <div data-cy="TabsComponent">
+      <div className="tabs is-boxed">
+        <ul>
+          {tabs.map(tab => (
+            <li
+              key={tab.id}
+              className={tab.id === currentTab.id ? 'is-active' : ''}
```
Comment (volodymyr-soltys97) [inline src/components/Tabs/Tabs.jsx:13]: Use the classnames library for add class with condition

File: src/components/Tabs/Tabs.jsx [hunk 6]
```diff
@@ -1 +1,36 @@
...
+export const Tabs = ({ tabs, activeTabId, onTabSelected }) => {
+  const currentTab = tabs.find(tab => tab.id === activeTabId) || tabs[0];
+
+  return (
+    <div data-cy="TabsComponent">
+      <div className="tabs is-boxed">
+        <ul>
+          {tabs.map(tab => (
+            <li
+              key={tab.id}
```
Comment (volodymyr-soltys97) [inline src/components/Tabs/Tabs.jsx:12]: Use destruction for `tab`

**Review by volodymyr-soltys97 — CHANGES_REQUESTED** [review]
Almost done!
Let's make your code better

**Review by volodymyr-soltys97 — APPROVED** [review]
Well done! 🔥

**Pull request closed without merging**